hack/actions/install-kubernetes-toolchain.sh

```diff
@@ -6,7 +6,7 @@ set -o pipefail
 
 readonly KUSTOMIZE_VERS="v3.5.4"
 readonly KUBECTL_VERS="v1.19.2"
-readonly KIND_VERS="v0.9.0"
+readonly KIND_VERS="v0.11.0"
```

Comment:
pending kind release, hopefully should be out soon

Reply:
@sunjayBhatia v0.10.0 is still the latest. Should this PR be marked "Draft" until v0.11.0 is released?

Reply:
sure, i thought 0.11.0 would be out by now

Reply:
might just put it back to installing the latest etc.

Reply:
its out now!